poc_iot_injector/src/receipt_txn.rs

```diff
@@ -54,6 +61,28 @@ pub fn handle_report_msg(
     })
 }
 
+/// Maybe squish poc_witnesses if the length is >= max_witnesses_per_receipt
+fn maybe_squish_witnesses(
+    poc_witnesses: &mut Vec<BlockchainPocWitnessV1>,
+    poc_id: &Vec<u8>,
+    max_witnesses_per_receipt: usize,
+) {
+    if poc_witnesses.len() <= max_witnesses_per_receipt {
+        return;
+    }
+
+    // Seed a random number from the poc_id for shuffling the witnesses
+    let mut hasher = Sha256::new();
```

Comment:
Use the Digest trait and the fact that GenericArray works for slices just fine:

```rust
    let seed = Sha256::digest(poc_id);
    let mut rng = StdRng::from_seed(seed.into());
```

Reply:
Oh nice, updated [6fc793e](https://github.com/helium/oracles/pull/291/commits/6fc793ede9854eecc9e7a197ad3c6675c1145e3d)